learn/tanssi/technical.md

```diff
@@ -3,51 +3,83 @@ title: Technical Features
 description: Tanssi is an Appchain protocol that eases the process of deploying Appchains so that developers can focus on their custom logic.
 ---
 
-# What is Tanssi? {: #what-is-tanssi } 
+# Technical features of Tanssi {: #technical-features-of-tanssi } 
 
-Tanssi is a protocol that makes deploying blockchains with custom logic specific to certain types of applications a breeze. These customized blockchains are normally referred to as Appchains, but in Tanssi terms, they are also known as Container Chains.
+## Introduction {: #introduction } 
 
-## The Problem
+As presented in the [Overview](/learn/tanssi/overview) article, Tanssi is an Appchain infrastructure protocol that streamlines the deployment of blockchains with custom logic specific to a wide range of use cases, including DeFi, NFTs, Gaming, and any other the developer teams may want to address.
 
-Appchains normally have to deal with the following problems:
+Infrastructure poses a huge challenge for developers, who would need to bootstrap collators for block production, data preservers, RPC endpoints, and deal with integrations and interoperability assigning precious effort and resources and losing focus on what is really important: the Appchain Runtime, the UX, and the value proposition to the users.
 
-- **Complex Infrastructure Management** - Appchain deployments typically demand handling numerous infrastructural components like block producers, validators, wallets, block explorers, indexers, RPC endpoints, and more. This is both time and resource consuming
+In Tanssi terms, Appchains are called ContainerChains, similar to the concept coined in [Docker](https://www.docker.com){target=blank}, allowing teams to focus on the product while alleviating deployment-related issues.
 
-- **Weak & Inefficient Security** - Appchains commonly suffer from having small or weak validator sets. Early-stage Appchains don't have enough economic guarantees to power a robust consensus mechanism. Moreover, paying for full blockchain capacity is inefficient when only a fraction of it is needed
+In this article, we discuss the following technical aspects of how Tanssi works:
 
-- **Cross-Chain and Interoperability** - Appchains inherently don't have cross-chain capabilities to connect to other blockchain ecosystems. Furthermore, developing interoperability solutions requires specialized expertise and meticulous implementation
+- **Collators Assignment**
+- **Block Production**
+- **Implementing Tanssi Protocol in your ContainerChain**
+- **Deploy a new ContainerChain**
```

Comment:
```suggestion
- **Collator assignment**
- **Block production**
- **Implementing the Tanssi protocol in your ContainerChain**
- **Deploying a new ContainerChain**
```

Reply:
I feel like the last two aren't really "how Tanssi works" but what is needed to integrate Tanssi - so maybe rephrase the descriptive line above this list to just be the technical aspects related to Tanssi